lib/ZnapZend/ZFS.pm

```diff
@@ -122,6 +123,25 @@ sub snapshotExists {
     return grep { $snapshot eq $_ } @snapshots;
 }
 
+sub createDataSet {
+    my $self = shift;
+    my $dataSet = shift;
+    my $remote;
+
+    ($remote, $dataSet) = $splitHostDataSet->($dataSet);
+    my @ssh = $self->$buildRemote($remote,
+        [@{$self->priv}, qw(zfs create), $dataSet]);
+
+    print STDERR '# ' . join(' ', @ssh) . "\n" if $self->debug;
+
+    $self->zLog->info("create new dataset ($dataSet) on destination ($remote)");
+
```

Comment:
hihi thats what you get when I comment on code that I did not write myself ... ok in that case